satpy/etc/readers/datatailor_nc.yaml

```diff
@@ -0,0 +1,21 @@
+reader:
+  name: datatailor_nc
+  short_name: EUMETSAT Data Tailor NetCDF
+  long_name:  Generic reader for simplified NetCDF files coming from the EUMETSAT Data Tailor
+  description: >
+    The EUMETSAT Data Tailor is a tool published by EUMETSAT that allows tailoring
+    satellite data to and from various file formats, among other things.
+    This is a generic reader for the simplified NetCDF output format.
+    For more information on the data tailor, see https://www.eumetsat.int/data-tailor
+  status: Alpha
+  supports_fsspec: True
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [many]
```

Comment:
I understand the idea here, but will this work in practice? Satpy has some parts dependent on the sensor name being correct (composites and enhancement). But maybe this is anyway superseded by the reader itself?

Reply:
I copied `many` from the `satpy_cf_nc` reader. I have no idea if it works.